paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSheetViewModelTest.kt

```diff
@@ -442,11 +446,15 @@ class CustomerSheetViewModelTest {
                 )
             )
         )
-        viewModel.result.test {
-            assertThat(awaitItem()).isNull()
-            viewModel.handleViewAction(CustomerSheetViewAction.OnPrimaryButtonPressed)
-            assertThat(awaitItem()).isInstanceOf(InternalCustomerSheetResult.Selected::class.java)
-        }
+        val viewStateTurbine = viewModel.viewState.testIn(viewModel.viewModelScope)
```

Comment:
I think we're missing some validation due to this change.

See the notes here: https://github.com/cashapp/turbine#multiple-flows

TLDR, we want to make sure the scope that we pass to `testIn` is going to complete, and I don't think we're completing the viewModelScope in these tests. It might be better to use `backgroundScope` here.